setup.py

```diff
@@ -186,7 +186,7 @@ def _main(argv):
     # tag = _run_cmd(["git", "describe", "--tags", "--exact-match", "@"])
 
     this_directory = Path(__file__).parent
-    long_description = (this_directory / "README.md").read_text()
```

Comment:
interesting :)

Reply:
mm I don't remember adding that, will revert it